script.js

```diff
@@ -179,4 +179,22 @@ openModal.forEach((element) => {
       });
     });
   });
-});
\ No newline at end of file
+});
+
+const email = document.getElementById('contact-email');
+const error = document.getElementById('error-message');
+const form = document.getElementById('form');
+
+error.style.visibility = 'hidden';
+
+function formValidate(event) {
+  if (email.value !== email.value.toLowerCase()) {
+    error.style.visibility = 'visible';
+    error.innerHTML = 'Please enter your email address in lowercase.';
+    event.preventDefault();
+  } else {
+    error.style.visibility = 'hidden';
+  }
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly use ES6 syntax as this is regarded as a good javascript practice.